pkg/webhook/webhook.go

```diff
@@ -0,0 +1,56 @@
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+type Webhook struct {
+	URL     string
+	Format  string
+	Timeout time.Duration
+}
+
+func (w *Webhook) Send(event trace.Event) error {
```

Comment:
Will this method make changes in the receiver in the future? If not, I suppose it can be a value receiver.